packages/datagateway-dataview/src/App.tsx

```diff
@@ -77,7 +77,36 @@ const registerRouteAction = {
     plugin: 'datagateway-dataview',
     displayName: 'DataGateway DataView',
     order: 0,
-    helpText: 'TODO: write some help text for the user tour',
+    helpSteps: [
+      {
+        target: '#plugin-link--browse-investigation',
+        content:
+          'DataGateway DataView allows you to browse datafiles, datasets and investigations using a variety of filters and sorting criteria',
+      },
+      {
+        target: '.tour-dataview-breadcrumbs',
+        content:
+          'The hierarchy of the currently displayed data is shown here, and allows navigation to the dataset or investigation containing it',
+      },
+      {
+        target: '.tour-dataview-results',
+        content:
+          'The number of results found taking into account any applied filters is displayed here',
+      },
+      {
+        target: '.tour-dataview-search-icon',
+        content: 'Navigate to Datagateway Search',
+      },
+      {
+        target: '.tour-dataview-cart-icon',
+        content:
+          'Displays the current number of items in the download cart, and navigates to Datagateway Download',
+      },
+      {
+        target: '.tour-dataview-card-view',
+        content: 'Toggle between list and card view',
```

Comment:
This help step does not get displayed, I think it is because the element does not have the class specified in `target`.

Reply:
Good catch, when I started this the card view hadn't been merged into master so I couldn't add the classname to the toggle as it wasn't there. But it's been merged now so I'll add that in.